dev.sparcles.SREngine.yml

```
@@ -0,0 +1,43 @@
+id: dev.sparcles.SREngine
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: srengine-run.sh
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home:create
```

```suggestion
  - --filesystem=home
```

Not sure where you found this, this has no meaning

---

yeah it was ignored a few times: https://github.com/flathub/flathub/pull/5482#discussion_r1704607353